extensions-contrib/opencensus-extensions/pom.xml

```diff
@@ -133,6 +142,18 @@
     </dependency>
   </dependencies>
   <build>
+    <pluginManagement>
+        <plugins>
+          <plugin>
+              <groupId>org.apache.maven.plugins</groupId>
+              <artifactId>maven-dependency-plugin</artifactId>
+              <configuration>
+              <!-- analyze incorrectly flags this dependency as missing when omitted, and unused when declared -->
+                <ignoredDependencies>io.grpc:grpc-protobuf:1.60.0</ignoredDependencies>
```

Comment:
I think we should use  <ignoredUnusedDeclaredDependency> to suppress the check similar to https://github.com/confluentinc/druid/blob/ea8baf1d6e021db292b0e6702641292e3b37639a/extensions-contrib/opentelemetry-emitter/pom.xml#L157C1-L157C1